modules/virt-about-vms-and-boot-sources.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-about-vms-and-boot-sources_{context}"]
+
+= About virtual machines and boot sources
+
+Virtual machines consist of a virtual machine definition and one or more disks, backed by datavolumes.
+
+Virtual machine templates enable you to create virtual machines using preset virtual machine definitions.
+
+Unlike a virtual machine template, a boot source is a fully configured virtual machine image that includes configured drivers. Each virtual machine template contains a virtual machine definition with a pointer to a boot source. For some templates, a boot source is automatically provided. If not, then the boot source must be prepared by an administrator.
+
+To use the boot sources feature, you must install the latest release of {VirtProductName}. The namespace `openshift-virtualization-os-images` enables the feature and is installed with the {VirtProductName} operator.
+
+After preparing and uploading boot source image files, you create configured virtual machines by attaching the boot source to a virtual machine template. The boot source is attached to a template using a `persistentvolumeclaim` (PVC) by uploading to a local file, cloning an existing PVC, or importing via registry or to a local file. Once the boot source is attached to a virtual machine template, any number of full-configured ready-to-use virtual machines can be created from the template.
+
+Once a virtual machine is created from a template, it is detached from the template. This process is known as _sealing_ a template.
```

Comment:
" This process is known as _sealing_ a template."

This is incorrect, we should drop it

Reply:
Done.